packages/main/src/StepInput.ts

```diff
@@ -122,6 +132,7 @@ type StepInputValueStateChangeEventDetail = {
 class StepInput extends UI5Element implements IFormInputElement {
 	eventDetails!: {
 		change: void
+		input: void
```

Comment:
I think the `inputType` event detail that exists on `InputEventDetail` is relevant for the user, so it must also be proxied.